src/Hocon.Configuration.Test/ConfigurationSpec.cs

```diff
@@ -31,19 +31,6 @@ public class MyObjectConfig
             public int[] IntergerArray { get; set; }
         }
 
-        [Fact]
-        public void Config_should_be_serializable()
```

Comment:
Fine deferring this for now

Reply:
This is moved to SerializationSpec. Config does not support direct JSON serialization anymore, because of immutability.